plugins/ruby-foreman-memcache/debian/gem.list

```diff
@@ -1,3 +1,3 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_memcache-0.1.0.gem"
+GEMS="https://rubygems.org/downloads/foreman_memcache-0.1.1.gem"
 GEMS="$GEMS https://rubygems.org/downloads/dalli-2.7.8.gem"
```

Comment:
I wonder if this needs to be bumped. Upstream 3.2 is out and the dependency is unbound, but on the other hand: the last release was in 2018. It wouldn't surprise me if most users have moved to the Redis caching backend that's in Foreman itself.

Should we merge this now just to get up to date?

Reply:
Given the path of update of foreman_memcache (automated tooling), and the fact that it at least looks like the inclusion of dalli 3.y would require an evaluation of backwards incompatible changes, I lean towards merging this as is.

Reply:
Me too. I started CI.